README.md

```diff
@@ -163,6 +163,119 @@ The script `load.py` (requires Python 3) can be used to restore from such JSON d
 
 The optional parameter `--no-commits` can be added to skip those entries that
 look like a commit ID. Note that this might break previous permalinks.
+
+# CONNECTING JAVA DEBUGGER WITH DOCKER COMPOSE
+
+Connecting to a java debugger can be broken down into the following steps:
+###1. Run the application using docker-compose
```

Comment:
Needs a blank line between `steps:` and `###`.  Needs a space after `###`
In other words:

```
Connecting to a java debugger can be broken down into the following steps:

### 1. Run the application using docker-compose
```